diff -ruN nant-0.92.orig/Makefile nant-0.92/Makefile
--- nant-0.92.orig/Makefile	2012-06-10 00:05:52.000000000 +1000
+++ nant-0.92/Makefile	2017-09-06 10:50:44.000000000 +1000
@@ -101,13 +101,13 @@
 	cp src/NAnt.Console/App.config bootstrap/NAnt.exe.config
 
 bootstrap/NAnt.Core.dll:
-	$(RESGEN)  src/NAnt.Core/Resources/Strings.resx bootstrap/NAnt.Core.Resources.Strings.resources
+	cp -f precompiled/NAnt.Core.Resources.Strings.resources bootstrap
 	$(MCS) $(DEBUG) -target:library -warn:0 -define:$(DEFINE) -out:bootstrap/NAnt.Core.dll -debug \
 		-resource:bootstrap/NAnt.Core.Resources.Strings.resources -r:lib${DIRSEP}common${DIRSEP}neutral${DIRSEP}log4net.dll \
 		-r:System.Web.dll -recurse:src${DIRSEP}NAnt.Core${DIRSEP}*.cs src${DIRSEP}CommonAssemblyInfo.cs
 
 bootstrap/NAnt.DotNetTasks.dll:
-	$(RESGEN)  src/NAnt.DotNet/Resources/Strings.resx bootstrap/NAnt.DotNet.Resources.Strings.resources
+	cp -f precompiled/NAnt.DotNet.Resources.Strings.resources bootstrap
 	$(MCS) $(DEBUG) -target:library -warn:0 -define:$(DEFINE) -out:bootstrap/NAnt.DotNetTasks.dll \
 		-r:./bootstrap/NAnt.Core.dll -r:bootstrap/lib/common/neutral/NDoc.Core.dll \
 		-recurse:src${DIRSEP}NAnt.DotNet${DIRSEP}*.cs -resource:bootstrap/NAnt.DotNet.Resources.Strings.resources \
diff -ruN nant-0.92.orig/src/NAnt.Core/NAnt.Core.build nant-0.92/src/NAnt.Core/NAnt.Core.build
--- nant-0.92.orig/src/NAnt.Core/NAnt.Core.build	2012-06-10 00:05:53.000000000 +1000
+++ nant-0.92/src/NAnt.Core/NAnt.Core.build	2017-09-06 10:51:05.000000000 +1000
@@ -23,9 +23,6 @@
                 <!-- common assembly-level attributes -->
                 <include name="../CommonAssemblyInfo.cs" />
             </sources>
-            <resources prefix="NAnt.Core" dynamicprefix="true">
-                <include name="Resources/**/*" />
-            </resources>
             <references>
                 <include name="${build.dir}/bin/log4net.dll"/>
                 <include name="System.Web.dll"/>
diff -ruN nant-0.92.orig/src/NAnt.DotNet/NAnt.DotNet.build nant-0.92/src/NAnt.DotNet/NAnt.DotNet.build
--- nant-0.92.orig/src/NAnt.DotNet/NAnt.DotNet.build	2012-06-10 00:05:53.000000000 +1000
+++ nant-0.92/src/NAnt.DotNet/NAnt.DotNet.build	2017-09-06 10:51:17.000000000 +1000
@@ -25,9 +25,6 @@
                 <include name="${nant::scan-probing-paths(build.dir + '/bin','NDoc.ExtendedUI.dll')}" />
                 <include name="${nant::scan-probing-paths(build.dir + '/bin','NDoc.Core.dll')}" />
             </references>
-            <resources prefix="NAnt.DotNet" dynamicprefix="true">
-                <include name="Resources/**/*" />
-            </resources>
         </csc>
     </target>
 </project>
